ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetCode.java

```diff
@@ -25,12 +25,23 @@
 
 import org.apache.tuweni.bytes.Bytes;
 
+/** The type Eth get code. */
```

Comment:
maybe it is worth changing these? idk
```suggestion
/** eth_getCode method. */
```